scripts/foundry/transmuter/TransmuterAddCollateralEURCV.s.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import "transmuter/transmuter/Storage.sol" as Storage;
+import { ITransmuter, ISettersGovernor, ISettersGuardian, ISwapper } from "transmuter/interfaces/ITransmuter.sol";
+import { Enum } from "safe/Safe.sol";
+import { MultiSend, Utils } from "../Utils.s.sol";
+import "../Constants.s.sol";
+
+contract TransmuterAddCollateralEURCV is Utils {
+    address public constant COLLATERAL_TO_ADD = 0x5F7827FDeb7c20b443265Fc2F40845B715385Ff2;
+    uint256 constant BPS = 1e14;
+
+    bytes oracleConfigCollatToAdd;
+    uint64[] public xFeeMint;
+    int64[] public yFeeMint;
+    uint64[] public xFeeBurn;
+    int64[] public yFeeBurn;
+    address public agToken;
+
+    function run() external {
+        uint256 chainId = vm.envUint("CHAIN_ID");
+
+        ITransmuter transmuter = ITransmuter(_chainToContract(chainId, ContractType.TransmuterAgEUR));
+        agToken = address(transmuter.agToken());
+        bytes memory transactions;
+        uint8 isDelegateCall = 0;
+        address to = address(transmuter);
+        uint256 value = 0;
+
+        uint64[] memory xFeeBurn = new uint64[](1);
+        uint64[] memory xFeeMint = new uint64[](3);
+        int64[] memory yFeeMint = new int64[](xFeeMint.length);
+        int64[] memory yFeeBurn = new int64[](xFeeBurn.length);
+        xFeeMint[0] = 0;
+        xFeeMint[1] = 0.29e9;
+        xFeeMint[2] = 0.30e9;
+        xFeeBurn[0] = 1e9;
+
+        yFeeBurn[0] = 0.9995e9; // 5BPS positive burning fee as we get rev share from it
```

Comment:
this is not how it works -> we need to put the real 5bps value here and not 1-5bps